dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/configuration/Configuration.kt

```diff
@@ -259,6 +261,18 @@ internal constructor(
             return this
         }
 
+        /**
+         * Add a network interceptor to the internal OkHttpClient used to upload tracked events to Datadog's servers.
+         * @param interceptor the interceptor to add
+         * Note you can call this method more than once if you need to add several interceptors
+         */
+        fun addNetworkInterceptor(interceptor: Interceptor): Builder {
```

Comment:
why not instead allowing a list of interceptors here ?

Reply:
Mostly because it mimicks the API used by OkHttp, where you add interceptors one at a time. 
Plus this is just for one customer, and we only expect them to submit a single interceptor, so it doesn't make sense to force them to create a list to do so.